pkg/operator/config.go

```diff
@@ -82,6 +98,25 @@ func getProviderControllerFromImages(platform configv1.PlatformType, images Imag
 	}
 }
 
+// When the platform type is BareMetal, MAO is also responsible for deploying
+// the baremetal-operator pod
+func getAdditionalProviderImages(platform configv1.PlatformType, images Images) (map[string]string, error) {
+        var baremetalImages map[string]string
```

Comment:
indentation is wrong all around. Please run `hack/go-fmt.sh`

Reply:
Please see response about indentation earlier.